datamodel/src/main/java/com/hartwig/serve/datamodel/gene/KnownGeneComparator.java

```diff
@@ -8,6 +8,11 @@ public class KnownGeneComparator implements Comparator<KnownGene> {
 
     @Override
     public int compare(@NotNull KnownGene gene1, @NotNull KnownGene gene2) {
-        return gene1.gene().compareTo(gene2.gene());
+        int geneCompare = gene1.gene().compareTo(gene2.gene());
```

Comment:
Java SDK offers some nice chaining of comparators:

```suggestion
        return Comparator.comparing(KnownGene::gene).thenComparing(KnownGene::geneRole).compare(gene1,
```

Reply:
Tempting but will leave as-is, there are a lot of comparators in SERVE and would rather refactor them in one go instead of having inconsistent ways